pkg/rbd/rbd_attach.go

```diff
@@ -282,16 +280,7 @@ func createPath(volOpt *rbdVolume, cr *util.Credentials) (string, error) {
 		klog.Warningf("rbd: map error %v, rbd output: %s", err, string(output))
 		return "", fmt.Errorf("rbd: map failed %v, rbd output: %s", err, string(output))
 	}
-	devicePath, found := waitForPath(volOpt.Pool, image, 10, useNBD)
-	if !found {
-		output, err := execCommand(cmdName, []string{
-			"unmap", imagePath, "--id", cr.ID, "-m", volOpt.Monitors, "--keyfile=" + cr.KeyFile})
-		if err != nil {
-			klog.Warningf("rbd: unmap error %v, rbd output: %s", err, string(output))
-		}
-		return "", fmt.Errorf("could not map image %s, Timeout after 10s", imagePath)
-	}
-	return devicePath, nil
+	return string(output), nil
```

Comment:
output will contain an extra newline at the end of the string, is this acceptable, or should we trim it?

Reply:
if it contains extra newline the CI should have failed isn't it?

Reply:
FWIW, I tested the output, before posting the comment, and it does contain an extra newline. The question is should we trim it or not. CI may not fail, but do we want the new line byte in the string that we are using, and hence may fail other comparisons down the line or is it better to be safe and store/return what we want without the new line?

Reply:
in that case, we can trim it, updated PR

Reply:
Thank you!